tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -691,6 +694,7 @@ def fit(self,
         ag_args_ensemble = kwargs['ag_args_ensemble']
         excluded_model_types = kwargs['excluded_model_types']
         use_bag_holdout = kwargs['use_bag_holdout']
+        calibrate = kwargs['calibrate']
```

Comment:
remove, use kwargs value directly

Reply:
pulled calibrate from kwargs

Reply:
Still need to remove, this variable is unused.

Reply:
removed